### PR TITLE
Fixed files copy/move when in favorites or recent section

### LINKED ISSUE #8851: Copymove has a question
### Steps to reproduce
1.Create a folder and create a file in the folder (optional), add a favorite to this file
image
2.In the Recent(Favorites) list, an error occurred while moving or copying the file
image
image
3.Copy is the same mistake

### Expected behaviour
Tell us what should happen
Can Copy/Move
### Actual behaviour
Tell us what happens instead
do not Copy/Move
### Server configuration

**Operating system**:

**Web server:**Nextcloud13

**Database:**MariaDB

**PHP version:**PHP 7.0.22-0ubuntu0.16.04.1

**Nextcloud version:** (see Nextcloud admin page)

**Updated from an older Nextcloud/ownCloud or fresh install:**fresh install

**Where did you install Nextcloud from:**github

**Signing status:**
<details>
<summary>Signing status</summary>

```
Login as admin user into your Nextcloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
```
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder

or 

Insert your config.php content here. 
Make sure to remove all sensitive content such as passwords. (e.g. database password, passwordsalt, secret, smtp password, …)
```
</details>

**Are you using external storage, if yes which one:** local/smb/sftp/...
NO
**Are you using encryption:** yes/no
No
**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
No
#### LDAP configuration (delete this part if not used)
<details>
<summary>LDAP config</summary>

```
With access to your command line run e.g.:
sudo -u www-data php occ ldap:show-config
from within your Nextcloud installation folder

Without access to your command line download the data/owncloud.db to your local
computer or access your SQL server remotely and run the select query:
SELECT * FROM `oc_appconfig` WHERE `appid` = 'user_ldap';


Eventually replace sensitive data as the name/IP-address of your LDAP server or groups.
```
</details>

### Client configuration
**Browser:**

**Operating system:**

#### Web server error log
<details>
<summary>Web server error log</summary>

```
Insert your webserver log here
```
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>

```
Insert your Nextcloud log here
```
</details>

#### Browser log
<details>
<summary>Browser log</summary>

```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```
</details>

### PR DESCRIPTION
**Pull request by skjnldsv**
Fix #8851

### CODE REVIEW

File: apps/files/js/fileactions.js [hunk 1]
```diff
@@ -644,10 +644,10 @@
 					}
 					OC.dialogs.filepicker(t('files', 'Target folder'), function(targetPath, type) {
 						if (type === OC.dialogs.FILEPICKER_TYPE_COPY) {
-							context.fileList.copy(filename, targetPath);
+							context.fileList.copy(filename, targetPath, false, context.dir);
 						}
 						if (type === OC.dialogs.FILEPICKER_TYPE_MOVE) {
-							context.fileList.move(filename, targetPath);
+							context.fileList.move(filename, targetPath, false, context.dir);
```
Comment (tcitworld) [inline apps/files/js/fileactions.js]: `move()` doesn't have a 4th parameter

Comment (skjnldsv) [reply apps/files/js/fileactions.js]: *facepalm*

Comment (skjnldsv) [reply apps/files/js/fileactions.js]: Don't judge me ^^'

Comment (tcitworld) [reply apps/files/js/fileactions.js]: Never occurred to me ;)

**Review by MorrisJobke — APPROVED** [review]
Tested and works 👍

File: apps/files/js/fileactions.js (hunk 1 above)
Comment (HandsomeBoby) [reply apps/files/js/fileactions.js]: @skjnldsv  The first time MoveCopy successfully . Error still occurs when the file is moved or copied a second time

Comment (skjnldsv) [reply apps/files/js/fileactions.js]: @HandsomeBoby a second time after reload? In the same location?

Comment (HandsomeBoby) [reply apps/files/js/fileactions.js]: @skjnldsv  Not understand what you mean. I mean, in the Recent, choose a file for MoveCopy is successfully,  reloading, there was an error while making a second MoveCopy to this file.
such as:
image
image  this is successfully! reloading
image
image  There was an error.

Comment (skjnldsv) [reply apps/files/js/fileactions.js]: Thanks! :)
let me try!

**Review by juliusknorr — APPROVED** [review]

File: apps/files/js/fileactions.js (hunk 1 above)
Comment (skjnldsv) [reply apps/files/js/fileactions.js]: Nope, can't reproduce with the last patch, it works great! :)
Let's get this in!

**Comment by skjnldsv** [comment]
Failure unrelated (swift) and checker pass on restart! :)
LGTM :rocket:

File: apps/files/js/fileactions.js (hunk 1 above)
Comment (HandsomeBoby) [reply apps/files/js/fileactions.js]: @skjnldsv 
Sorry,My English is a bit bad.
On the recent page,move a file in the folder to the “/”, an error occurred while moving this file again.  Not need to reload.

**Comment by MorrisJobke** [comment]
@skjnldsv What about the backport?

**Comment by MorrisJobke** [comment]
> @skjnldsv What about the backport?

Ping ;)

**Pull request merged**